### DISCUSSION TITLE
Not sure how to add Hubitat ip etc.

### DISCUSSION THREAD

**Discussion by stampeder** [jason0x43/hacs-hubitat#251] [General]
I first tried HA back in 2019 and got totally frustrated. I just came back and spun up HA on my Intel NUC and I must say I am extremely impressed. I have two Hubitat C7s and I am trying to integrate one of them into HA. I have HACS integrated. I have been following https://github.com/jason0x43/hacs-hubitat install instructions. My issue is I don't understand how to set the following: Pick “Hubitat”, then provide:

    The address of the hub (e.g., http://10.0.1.99 or just 10.0.1.99 if you’re not using https)
    The app ID of the Maker API instance (the 2, 3 or 4 digit number after /apps/api/ in any of the Maker API URLs)
    The API access token
    Optional: An address for the event server to listen on (more about this below); this will be chosen automatically by default
    Optional: A port for the event server to listen on (more about this below); this will be chosen automatically by default
    Optional: Provide the relative paths to an SSL private key and certificate (e.g., ssl/localhost-key.pem and ssl/localhost.pem). These are files that you will need to generate using a tool such as mkcert or openssl If these paths are provided, the event server (described below) will serve over SSL (and only over SSL).   Jason says to pick "Hubitat" however nothing comes up to query the above settings. I have rebooted HA.

**Comment by jason0x43** [comment]
There are two steps to using an integration. First, you have to install the integration so that HA can use it. Then, you configure an instance of the integration.

The easiest way to manage custom integrations is through [HACS](https://hacs.xyz/docs/setup/download/). If you install HACS, Hubitat is included in the built-in list of available integrations to install through HACS. 

Once you've installed the Hubitat integration and restarted HA, you can go into Settings -> Integrations and click the '+ Add Integration' button to add an instance of the Hubitat integration. Pick Hubitat from the list, and you can then enter the information you mentioned.

**Comment by stampeder** [comment]
Argghhhh!! I missed the second "+Add Integration" in the Setting Integration. I have done it and now my integration has come up. Thanks for the help!!! Still a newbie so still climbing the learning curve.
